aws/aws-sdk-kotlin
# Update codegen to generate base exception rather than UnknownServiceErrorException

## Issue \#

https://github.com/awslabs/aws-sdk-kotlin/issues/458

## Description of changes

* Updates codegen to produce the exception type as specified in #458 

## Additional Testing
* Manually verify S3 exception codegen
* Verify all services compile locally after change

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
